src/nvim/buffer.c

```diff
@@ -2770,6 +2770,16 @@ void free_titles(void)
 
 # endif
 
+/**
+ * Enumeration specifying the valid numeric bases that can
+ * be used when printing numbers in the status line.
+ **/
```

Comment:
Style guide accepts *only* `//`-style comments (specifically `///` for doxygen). `/*…*/` would be fine only inside a macros and only as long as they are one-line.